cfn-resources/project-invitation/test/inputs_1_invalid.template.json

```diff
@@ -1,8 +1 @@
-{
```

Comment:
delete file?

Reply:
we don't use invalid template in the cfn tests, we should delete all of them

Reply:
right, so we should delete this one in this PR right? currently this is an empty file

Reply:
oh 🤦 thanks.  I will remove it xD

Reply:
updated

Reply:
Small question:

> we don't use invalid template in the cfn tests

Is this something particular to how cfn runs contract tests, or a decision that was made from our side?

Reply:
The invalid template was added initially by contractors but then they figured out that was not used by the test scenarios run as part of cfn test https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/contract-tests.html